operations/iscsi_sbps/iscsi_sbps.md

```diff
@@ -265,15 +265,28 @@ Node personalization is the prerequisite step of SBPS solution where we need to
 with necessary provisioning, configuration and enable required components. The required RPMs for `targetcli` command / LIO are part of NCN node image in CSM 1.6.
 The SBPS Marshal Agent gets installed during node personalization using CFS.
 
-In CSM 1.6.0, all worker nodes are configured as iSCSI targets by default. In CSM 1.7.0, selective worker node personalization is introduced and steps for the same for different scenarios is as below:
+In CSM 1.6.0, all worker nodes are configured and enabled as iSCSI SBPS targets during worker node personalization.
+
+In CSM 1.7.0, selective iSCSI worker node personalization is introduced. All worker nodes are still **configured** for
+iSCSI, but selective node personalization gives administrators control over which worker nodes are enabled as iSCSI SBPS
+targets. The default behavior is still the same as in CSM 1.6.0, so if no action is taken to use this feature, then all
+worker nodes will continue to be enabled as iSCSI targets.
+
+Selective iSCSI worker node personalization uses an HSM group named `iscsi_worker` as the mechanism for an administrator
+to specify which workers should be enabled as iSCSI targets. If this group does not exist, then all worker nodes will be
+enabled as iSCSI targets. While it is technically possible to create this group and leave it empty, this will mean that
+no worker nodes will be enabled as iSCSI targets, which in turn will mean that no managed nodes will be able to boot.
+Therefore CSM generally treats it as an error if the HSM group exists but it contains no worker nodes.
+
+For details on creating this group, see [Managing selective node personalization](../iscsi_sbps/Managing_selective_node_personalization.md) and this has to be done in certain stages in below scenarios to avail this feature:
 
 1) Fresh Install:
 
      1.1 Create HSM group
 
      1.2 Enable/ Configure iSCSI SBPS
```

Comment:
What is this list? If I was an admin reading this, I'd assume these are steps I have to follow. But step 1 is only needed if I want to use that feature. And step 2 isn't anything I do explicitly. Some text needs to be added before these to provide context.

Reply:
Talked to Ashalatha. We've agreed that this part of the `iscsi_sbps.md` page is going to be covered by the revisions to the "Manage selective node personalization" page. So I'm going to mark this comment resolved. I'll be reviewing the updates made to the other page, so if there are problems after the move, I can make new comments.